.github/ISSUE_TEMPLATE/golang_version_upgrade.md

```diff
@@ -0,0 +1,87 @@
+    
+    `find build -type f \( -not -path '*/\.*' -and -not -path 'build/tmp/*' \) -exec sed -i 's/GO_VERSION=[0-9]\+\.[0-9]\+\.[0-9]\+/GO_VERSION=<NEW_GOLANG_VERSION>/g' {} \;`
+- [ ] Update `go.mod` files for the `build` directory:
+    Look for the `go.mod` files in the sub-directories of `build/script`, for each of them, run:
+
+    `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+- [ ] Update the Go version for each example that was written in Golang:
+    - [ ] Update Dockerfiles for `examples` directory:
+
+        At the `examples` directory, run:
+        
+        `find examples -name Dockerfile -exec sed -i 's/golang:[0-9]\+\.[0-9]\+-alpine/golang:<NEW_GOLANG_VERSION_WITHOUT_PATCH>-alpine/g' {} \;`
+
+        At the `examples` directory, run:
+        
+        `find examples -name Dockerfile -o -name Dockerfile.windows -exec sed -i 's/golang:[0-9]\+\.[0-9]\+\.[0-9]\+/golang:<NEW_GOLANG_VERSION>/g' {} \;`
+    - [ ] Update `go.mod` files for `examples` directory and update the example image tag:
+        - [ ] In `examples/allocation-endpoint/client`, run:
+        
+            `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+
+        - [ ] In `examples/allocation-endpoint/server`, run:
+
+            `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+
+        - [ ] In `examples/allocation-webhook`:
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/allocation-webhook/Makefile`, increase the version number by 1 for `autoscaler-webhook`
+
+        - [ ] In `examples/crd-client`:
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/crd-client/Makefile`, increase the version number by 1 for `crd-client`
+
+        - [ ] In `examples/custom-controller`
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/custom-controller/Makefile`, increase the version number by 1 for `custom-controller`
+
+        - [ ] In `examples/simple-game-server`
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/simple-game-server/Makefile`, increase the version number by 1 for `simple-game-server`
+
+        - [ ] In `examples/simple-genai-server`
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/simple-genai-server/Makefile`, increase the version number by 1 for `simple-genai-game-server`
+
+        - [ ] In `examples/supertuxkart`
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/supertuxkart/Makefile`, increase the version number by 1 for `supertuxkart-example`
+
+        - [ ] In `examples/xonotic`
+            - [ ] Run `go mod edit -go=<NEW_GOLANG_VERSION> && go mod tidy`
+            - [ ] In `examples/xonotic/Makefile`, increase the version number by 1 for `xonotic-example`
```

Comment:
We can do this:
1) Navigate to examples directory
2) Run `find . -name 'go.mod' -execdir go mod edit -go=<NEW_GOLANG_VERSION> \;`
3) Run `find . -name 'go.mod' -execdir go mod tidy \;`

WDYT?

Reply:
We have a script to increment the examples image in all the files[not only /examples] https://github.com/googleforgames/agones/blob/main/build/scripts/bump-image/main.go#L15
 
Navigate to `/build` and run `make bump-image IMAGENAME=<imageName> VERSION=<current-version>`